app/models/header_lookup.rb

```diff
@@ -43,7 +43,7 @@ def mesh_term_pid_lookup(mesh_term="")
   # lookup lcsh term, memoize it, write it to file, return PID
   def lcsh_term_pid_lookup(lcsh_term="")
     stripped_lcsh_term = strip_accents(lcsh_term)
-    subject_names, subject_id_uris = perform_and_parse_lcsh_query(stripped_lcsh_term)
+    subject_names, subject_id_uris = perform_and_parse_lcsh_query(CGI.escape(stripped_lcsh_term))
```

Comment:
I think that the same string, `Obesity-- prevention & control` appears in the LCSH terms as well so I figured it made sense to change it here as well.